person/views.py

```diff
@@ -35,34 +37,64 @@ class PersonDetail(DetailView):
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
 
+        # NOTE: this sort does not work as expected...
         last_cited_attr = '-object_ref__membershippersonlastciteddate'
-        memberships = context['person'].membershippersonmember_set\
-                        .order_by(last_cited_attr)
+        # memberships = context['person'].membershippersonmember_set\
+        #                 .order_by(last_cited_attr)
 
-        # Start by getting the most recent membership for the "Last seen as"
-        # description
+
+        # Use SQL...
```

Comment:
This would be a good opportunity to drop a short comment about why we decided to go with SQL instead of using the ORM. Might be helpful to us if we need to clean up the code later.